tests/ledger.py

```diff
@@ -0,0 +1,158 @@
+import io
```

Comment:
Copyright missing

Reply:
The script should catch that, if not, it needs the re-configured to pick up this file too.